openwpm/mp_logger.py

```diff
@@ -16,7 +16,7 @@
 from sentry_sdk.integrations.logging import BreadcrumbHandler, EventHandler
 from tblib import pickling_support
 
-from .Commands.utils.webdriver_utils import parse_neterror
+from .commands.utils.webdriver_utils import parse_neterror
 from .socket_interface import serversocket
```

Comment:
Can you please turn `serversocket` into `ServerSocket`. Sorry for missing this before.